---
int16 status
Log[] logs